ros_buildfarm/templates/release/trigger_upload_repo_job.xml.em

```diff
@@ -40,6 +40,7 @@
 $HOME/upload_triggers/upload_repo.bash @target
 @[end for]@
 </command>
+      <configuredLocalRules/>
```

Comment:
* hudson.tasks.Shell added the ability to filter or reset environment
variables from specific steps and the default configuration saves a
self-closing configuredLocalRules tag which contains that information.